build_helpers/publishPackage.sh

```diff
@@ -60,14 +95,21 @@ git commit -m "Version $next_version"
 
 next_ref="v$next_version"
 
+echo "Tagging $next_version"
 git tag "$next_ref"
 git push origin "$next_ref"
 
-if [ $BETA = 1 ]; then
+echo "Pushing release cut"
+git push origin $BRANCH
+
+if [ $BETA = true ]; then
+  echo "Publishing beta tag"
   npm publish --tag beta
-else
+elif [ $LATEST = true ]; then
+  echo "Tagging" $next_ref "as 'latest'"
   git tag latest -f
   git push origin latest -f
-  git push origin master
-  npm publish
+
+  echo "Publishing latest tag"
+  npm publish --tag latest
```

Comment:
Not passing `--tag` defaults it to `latest`. But I feel passing it explicitly is clearer.